packages/harmony/src/storybook/components/ComponentRules.tsx

```diff
@@ -9,13 +11,13 @@ type ComponentRulesProps = {
     positive: {
       component: ReactElement
       description: string | ReactElement
-      style?: CSSProperties
+      css?: CSSObject
```

Comment:
what is CSSProperties vs CSSObject?

Reply:
CSSObject is what is allowed for the css prop basically, not sure of its actual differences though, i think its a superset, cause it also allows for nested selectors etc